testplan/base.py

```diff
@@ -379,11 +391,26 @@ def run(self):
         except Exception:
             pass
 
-        result = super(Testplan, self).run()
-        if isinstance(result, TestRunnerResult):
-            testplan_result = TestplanResult()
-            testplan_result.__dict__ = result.__dict__
-            return testplan_result
+        with tracing.conditional_span(
+            self.cfg.name,
```

Comment:
we are going to create the span only if we have self.cfg.name? i don't quite understand why